hack/release.sh

```diff
@@ -65,28 +67,26 @@ ko resolve ${KO_FLAGS} -f config/ >> "${SERVING_YAML}"
 
 echo "Building Monitoring & Logging"
 # Use ko to concatenate them all together.
-ko resolve ${KO_FLAGS} -R -f config/monitoring/100-common \
-    -f config/monitoring/150-elasticsearch \
-    -f third_party/config/monitoring/common \
-    -f third_party/config/monitoring/elasticsearch \
-    -f config/monitoring/200-common \
-    -f config/monitoring/200-common/100-istio.yaml >> "${MONITORING_YAML}"
-
-# Use ko to do the same for the lite version.
-ko resolve ${KO_FLAGS} -R -f config/monitoring/100-common \
-    -f third_party/config/monitoring/common/istio \
-    -f third_party/config/monitoring/common/kubernetes/kube-state-metrics \
-    -f third_party/config/monitoring/common/prometheus-operator \
-    -f config/monitoring/150-elasticsearch/100-scaling-configmap.yaml \
-    -f config/monitoring/200-common/100-fluentd.yaml \
-    -f config/monitoring/200-common/100-grafana-dash-knative-efficiency.yaml \
-    -f config/monitoring/200-common/100-grafana-dash-knative.yaml \
-    -f config/monitoring/200-common/100-grafana.yaml \
-    -f config/monitoring/200-common/100-istio.yaml \
-    -f config/monitoring/200-common/200-prometheus-exporter \
-    -f config/monitoring/200-common/300-prometheus \
-    -f config/monitoring/200-common/100-istio.yaml >> "${MONITORING_LEAN_YAML}"
-
+ko resolve ${KO_FLAGS} -R -f config/monitoring/100-namespace.yaml \
+    -f third_party/config/monitoring/logging/elasticsearch \
+    -f config/monitoring/logging/elasticsearch \
+    -f third_party/config/monitoring/metrics/prometheus \
+    -f config/monitoring/metrics/prometheus \
+    -f config/monitoring/tracing/zipkin >> "${MONITORING_YAML}"
+
+# Metrics via Prometheus & Grafana
+ko resolve ${KO_FLAGS} -R -f config/monitoring/100-namespace.yaml \
+    -f third_party/config/monitoring/metrics/prometheus \
+    -f config/monitoring/metrics/prometheus >> "${MONITORING_METRIC_PROMETHEUS_YAML}"
+
+# Logs via ElasticSearch, Fluentd & Kibana
+ko resolve ${KO_FLAGS} -R -f config/monitoring/100-namespace.yaml \
+    -f third_party/config/monitoring/logging/elasticsearch \
+    -f config/monitoring/logging/elasticsearch >> "${MONITORING_LOG_ELASTICSEARCH_YAML}"
+
+# Traces via Zipkin
+ko resolve ${KO_FLAGS} -R -f config/monitoring/100-namespace.yaml \
```

Comment:
I don't think the `100-namespace.yaml` is needed here since both resources are create in the `istio-system` namespace

Reply:
Good catch. Will fix.